client/templates/frontlayout.html

```diff
@@ -0,0 +1,7 @@
+<template name="frontlayout">
+  <div>
+    {{> header}}
+
+    {{> frontbody}}
```

Comment:
@JuliaKoe This should have been a template dynamic so that the "main" parameter can be passed in by flow layout!